/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
/shared/centos7/python/3.8.1/bin/python3: can't open file 'run.py': [Errno 2] No such file or directory
srun: error: c0429: tasks 0-27: Exited with exit code 2
